### PR TITLE
Allow any param

### PR DESCRIPTION
**Pull request by caub**
Allow to set max param for a pool, from querystring

### CODE REVIEW

**Comment by monteslu** [comment]
@caub looks good, can you fix conflicts, and I'll get it merged

File: test/parse.js [hunk 1]
```diff
@@ -182,9 +196,11 @@ test('do not override a config field with value from query string', function(t){
...
 
-test('return last value of repeated parameter', function(t){
-  var connectionString = 'pg:///?keepalives=1&keepalives=0';
-  var subject = parse(connectionString);
-  t.equal(subject.keepalives, '0');
-  t.end();
-});
\ No newline at end of file
+
+// test('return last value of repeated parameter', function(t){
```
Comment (caub) [inline test/parse.js]: @monteslu is it really necessary this test and behavior? 

I think it's more confusing, and also I'd prefer to use URL and URL#searchParams#get (but it takes first value not last)

**Comment by mariusa** [comment]
Why just max? All config parameters should work, eg ssl

**Comment by caub** [comment]
Renamed, this PR allows anything

**Comment by caub** [comment]
@monteslu is it ok like this?

**Comment by caub** [comment]
tell me if anything needs to be changed

**Comment by caub** [comment]
@monteslu Is the PR ok? or I close it?

it's like the master branch, just more simple

**Pull request merged**